tests/test_prep.py

```diff
@@ -58,50 +58,54 @@ def test_get_candidate_words():
 
 @patch('topicexplorer.prep.input')
 def test_get_high_filter(input_mock):
```

Comment:
There will be some revisions to these tests based on changes to the `topicexplorer.prep.get_high_filter_stops` call signature - namely, it should not return a third part of the tuple called `valid` - rather a `ValueError` exception should be raised and handled by the caller.